### PR TITLE
BaSP-MR-53: Inputs components

### PR DESCRIPTION
**Pull request by Bigzufi**
Three types of inputs was created
-Input: It's a regular input that can be used as a text input or a password input through the prop "type"
-Textarea
-DatePicker

### CODE REVIEW

File: src/Components/Shared/Inputs/DatePicker.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './inputs.module.css';
+
+const DatePicker = ({ value, onChangeDate }) => {
```
Comment (Dannylez) [inline src/Components/Shared/Inputs/DatePicker.jsx]: should be great to have props for "name" and "label"

File: src/Components/Shared/Inputs/TextArea.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './inputs.module.css';
+
+const TextArea = ({ value, onChangeArea, placeholder }) => {
```
Comment (Dannylez) [inline src/Components/Shared/Inputs/TextArea.jsx]: should be great to have props for "name" and "label"

**Review by Dannylez — APPROVED** [review]

File: src/Components/Shared/Inputs/DatePicker.jsx (hunk 1 above)
Comment (Bigzufi) [reply src/Components/Shared/Inputs/DatePicker.jsx]: 👍

File: src/Components/Shared/Inputs/TextArea.jsx (hunk 2 above)
Comment (Bigzufi) [reply src/Components/Shared/Inputs/TextArea.jsx]: 👍

**Review by DelfiLT — APPROVED** [review]

**Pull request merged**